cli/changelog/0.74.2.md

```diff
@@ -0,0 +1,3 @@
+### Fixes
+
+- Don't reject the `runtime` experimental feature flag added in SDK 0.25.0
```

Comment:
In a way this is a feature, but it's not really a feature of the CLI so I've labelled it as a fix.  Feel free to tell me I'm wrong though